### PR TITLE
Change tab close and rename icons to better fit the UI

### LINKED ISSUE #8419: Tab close icon is too big
# Environment

```none
Windows build number: 10.0.19042.630
Windows Terminal version (if applicable): 1.4.3243.0

Any other software? No
```

# Steps to reproduce

Right-click on an open tab, notice the close option

# Expected behavior

The tab close icon should be the same size as the other icons

# Actual behavior

The tab close icon is too large and distracting almost.

image

### PR DESCRIPTION
**Pull request by Hegunumo**
## Summary of the Pull Request
Made the tab icon smaller so that it is the same size as the other icons
 
## PR Checklist
* [x] Closes #8419 
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Documentation updated. If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/terminal) and link it here: #xxx
* [ ] Schema updated.
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

File: src/cascadia/TerminalApp/Tab.cpp [hunk 1]
```diff
@@ -526,7 +526,7 @@ namespace winrt::TerminalApp::implementation
         Controls::MenuFlyoutItem closeTabMenuItem;
         Controls::FontIcon closeSymbol;
         closeSymbol.FontFamily(Media::FontFamily{ L"Segoe MDL2 Assets" });
-        closeSymbol.Glyph(L"\xE8BB");
+        closeSymbol.Glyph(L"\xEDAE");
```
Comment (KalleOlaviNiemitalo) [inline src/cascadia/TerminalApp/Tab.cpp:529]: [Segoe MDL2 icons](https://docs.microsoft.com/en-us/windows/uwp/design/style/segoe-ui-symbol-font#icon-list) mentions "EDAE ErrorBadge12". Using an error badge to represent closing a tab seems dubious to me, even if the glyph is currently similar.

Comment (Hegunumo) [reply src/cascadia/TerminalApp/Tab.cpp:529]: Then maybe I will try make the original icon smaller > <

**Comment by Hegunumo** [comment]
I can't change the size using `Fontsize()`

**Comment by zadjii-msft** [comment]
@Hegunumo Could you elaborate a bit more? Without actually checking if this worked, the following does seem to at least compile for me...
image

**Comment by Hegunumo** [comment]
Yes it compiles, but no matter what integer I put to parentheses size of glyph  won't change at all. I can't make it either smaller or bigger.

File: src/cascadia/TerminalApp/Tab.cpp (hunk 1 above)
Comment (EOussama) [reply src/cascadia/TerminalApp/Tab.cpp:529]: Either that or use “F13D StatusCircleErrorX”

**Review by EOussama — COMMENTED** [review]
`F13D` (StatusCircleErrorX)is a much smaller variant of the icon. It could work.

File: src/cascadia/TerminalApp/Tab.cpp [hunk 2]
```diff
@@ -572,7 +572,7 @@ namespace winrt::TerminalApp::implementation
             // "Rename Tab"
             Controls::FontIcon renameTabSymbol;
             renameTabSymbol.FontFamily(Media::FontFamily{ L"Segoe MDL2 Assets" });
-            renameTabSymbol.Glyph(L"\xE932"); // Label
+            renameTabSymbol.Glyph(L"\xE8AC"); // Label
```
Comment (zadjii-msft) [inline src/cascadia/TerminalApp/Tab.cpp:575]:
```suggestion
            renameTabSymbol.Glyph(L"\xE8AC"); // Rename
```

**Review by zadjii-msft — APPROVED** [review]
| before | after | 
| -- | -- |
| image | image |

Definitely looks better to me, thanks!

**Comment by carlos-zamora** [comment]
I'm ok with this. Just fix the merge conflicts and I can approve/merge it. 😊

**Comment by DHowett** [comment]
@carlos-zamora you can definitely approve something that has merge conflicts -- it _cannot_ be merged even with your approval if there are conflicts.

**Review by carlos-zamora — APPROVED** [review]

**Review by DHowett — APPROVED** [review]

**Comment by DHowett** [comment]
thank you!

**Pull request merged**